### ISSUE TITLE
USWDS-Compile - Feature: Configurable browserslist settings

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds-compile#86]
### Is your feature request related to a problem? Please describe.

Customizing the browsers that are supported would allow teams to adjust based on their project needs.

### Describe the solution you'd like

Users should be allowed to customize or override browserslist settings with their own configuration.

### Describe alternatives you've considered

n/a

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-compile/issues) for duplicate feature requests.

**Comment by mdmower-csnw** [comment]
@mejiaj - beyond the already supported [`settings.compile.browserslist`](https://github.com/uswds/uswds-compile?tab=readme-ov-file#additional-settings) that affects autoprefixer, what else were you thinking this could do?

**Comment by mejiaj** [comment]
@mdmower-csnw you're right, this is a redundant issue and closing. Thanks for flagging.
